conf/arm.src

```diff
@@ -1,6 +1,6 @@
-SOURCE_URL=https://github.com/SmartHoneybee/ubiquitous-memory/releases/download/v5.31.0/mattermost-v5.31.0-linux-arm.tar.gz
-SOURCE_SUM=15af0f4757fe1db9f6322fd9d5551aacbfe1c4082ad543dc4641d0a541133b5f
-SOURCE_SUM_PRG=sha256sum
+SOURCE_URL=https://github.com/SmartHoneybee/ubiquitous-memory/releases/download/v5.31.0/mattermost-v5.31.1-linux-arm.tar.gz
+SOURCE_SUM=00851098feb54afbe457c9991dcf549aa8a4967aecd80bd489865b8fe3afcde02a150b45b1c2b5aec2b02d67543f186be9a245894983270f521e18e464d947d9
+SOURCE_SUM_PRG=sha512sum
```

Comment:
@ericgaspar will putting the sha512 and using `sha512sum` work properly? (Or should we recompute a sha256 digest every time?)

Reply:
yes, sha512 should work